tests/src/deviceLib/hipSimpleAtomicsTest.cpp

```diff
@@ -292,5 +293,5 @@ void runTest(int argc, char** argv) {
     free(hOData);
     hipFree(dOData);
 
-    passed();
+    //passed();
```

Comment:
The right way is to conditionally execute passed() based on the value of testResult.